pulsar-websocket/src/main/java/org/apache/pulsar/websocket/service/WebSocketProxyConfiguration.java

```diff
@@ -72,12 +72,20 @@
     @FieldContext(doc = "Connection string of configuration metadata store servers")
     private String configurationMetadataStoreUrl;
 
-    @FieldContext(doc = "ZooKeeper session timeout in milliseconds")
+    @FieldContext(doc = "ZooKeeper session timeout in milliseconds. "
+            + "@deprecated - Use metadataStoreSessionTimeoutMillis instead.")
     private long zooKeeperSessionTimeoutMillis = 30000;
 
-    @FieldContext(doc = "ZooKeeper cache expiry time in seconds")
+    @FieldContext(doc = "ZooKeeper cache expiry time in seconds. "
+            + "@deprecated - Use metadataStoreCacheExpirySeconds instead.")
     private int zooKeeperCacheExpirySeconds = 300;
```

Comment:
```suggestion
    @Deprecated
    @FieldContext(doc = "ZooKeeper cache expiry time in seconds. "
            + "@deprecated - Use metadataStoreCacheExpirySeconds instead.",
            deprecated = true)
    private int zooKeeperCacheExpirySeconds = 300;
```
It's better to add `deprecated` here.